### ISSUE TITLE
Accessibility issue with underlined text in rules pages

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules-web#167]
The rules pages use the `<u>` element for definitions of key terms. This means users need colour vision to tell key terms and links apart:

https://act-rules.github.io/rules/fd3a94#equivalent-resource

```
<a id="equivalent-resource" href="#equivalent-resource">
<h3>Equivalent resource</h3>
</a>
<i>key: <u>equivalent-resource</u></i>
```

**Comment by WilcoFiers** [comment]
Don't think that's a failure of WCAG, but it isn't great. IMO we should just remove the key thing all together. It's only useful for rule authors, and they have other ways to get at that info too.

**Comment by jkodu** [comment]
Fixed in #172
